functions/ts/kubeval/src/kubeval.ts

```diff
@@ -225,21 +225,29 @@ function readStdoutToString(childProcess: ChildProcess): Promise<string> {
 }
 
 kubeval.usage = `
-Validates configuration using kubeval.
+Validate KRM resources using kubeval.
```

Comment:
are you duplicating the README here? is there an issue for having one source of truth?

Reply:
Manually in this PR.
We are tracking it in https://github.com/GoogleContainerTools/kpt/issues/1641.